Source/SwiftLintFramework/Rules/FirebaseConfigActivateRule.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  FirebaseConfigActivateRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigActivateRule: ASTRule, ConfigurationProviderRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_activate",
+        name: "Firebase Config Activate",
+        description: "Firebase Config should be activated.",
+        nonTriggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n self.remoteConfig.activateFetched() \n }"
+        ],
+        triggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n }"
+        ]
+    )
```

Comment:
What if the codebase has an instance method called .fetch which is not related to Firebase? By the code below it seems a violation would be triggered.